### PR TITLE
Fix PointEnv

### PR DESCRIPTION
**Pull request by CatherineSue**
PointEnv didn't update its point location, which makes the environment
not step and always return constant reward.

### CODE REVIEW

File: src/garage/envs/point_env.py [hunk 1]
```diff
@@ -27,10 +27,12 @@ def __init__(
             goal=np.array((1., 1.), dtype=np.float32),
             done_bonus=0.,
             never_done=False,
+            action_scale=1.,
```
Comment (ryanjulian) [inline src/garage/envs/point_env.py]: please don't scale actions inside environments. it's up to the algorithm to do this.

**Comment by ryanjulian** [comment]
@ahtsan feel free to merge this when you're happy with it

File: src/garage/envs/point_env.py [hunk 2]
```diff
@@ -79,6 +79,7 @@ def step(self, action):
         a = action.copy()  # NOTE: we MUST copy the action before modifying it
         a = np.clip(a, self.action_space.low, self.action_space.high)
 
+        self._point = np.clip(self._point + a, -5, 5)
```
Comment (ryanjulian) [inline src/garage/envs/point_env.py:82]: perhaps define `ARENA_SIZE = 5` at the top of the file to make for easier maintenance.

**Comment by naeioi** [comment]
Add some tests. Please take a second look.

**Pull request merged**